src/ripple/app/ledger/impl/LedgerCleaner.cpp

```diff
@@ -301,7 +311,8 @@ class LedgerCleanerImp
 
         if(! getApp().getLedgerMaster().fixIndex(ledgerIndex, ledgerHash))
         {
-            m_journal.debug << "ledger " << ledgerIndex << " had wrong entry in history";
+            m_journal.debug << "ledger " << ledgerIndex
+                            << " had wrong entry in history";
```

Comment:
It is subjective.  I am biased as a left-to-right reader.  My eyes tend to see the left of a line first, and so I like to put the most important information first.  To me it is very important to recognize that this is a continuation line, and what it is a continuation of.  I find lines that start with `<<`, where that `<<` is vertically lined up with a previous `<<` quite indicative of a multi-line streaming operation.

I do not feel strongly enough about this style to mandate it across the project.  I think having a somewhat permissive style guide is good in that it allows us to experiment with readability such as you have been doing with more frequent line breaks.  As long as the end result is readable (a very subjective term), I think we should allow it.